server/rest/src/test/java/org/infinispan/rest/resources/CacheResourceV2Test.java

```diff
@@ -475,6 +476,52 @@ public void testCacheV2Distribution() {
       }
    }
 
+   @Test
+   public void testCacheV2KeyDistribution() {
+      final String cacheName = "keyDistribution";
+      String cacheJson = "{ \"distributed-cache\" : { \"statistics\":true } }";
+      createCache(cacheJson, cacheName);
+
+      RestCacheClient cacheClient = client.cache(cacheName);
+
+      putStringValueInCache(cacheName, "key1", "data");
+      putStringValueInCache(cacheName, "key2", "data");
+      Map<String, Boolean> sample = new HashMap<>() {{
```

Comment:
can just use Map.of instead now.

`Map.of("key1", true, "key2", true, "unknown", false);`